CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 - metrics-netstat-tcp.rb: Option to disable IPv6 check (#44 via @MattMencel)
 - check-multicast-groups.rb: Fix undefined method deep_merge for []:Array (@maoe)
 - check-whois-domain-expiration-multi.rb does not fail as timeout is cast to integer (@majormoses)
+- check-multicast-groups.rb: Stop using settings['check-multicast-groups'] (@maoe)
```

Comment:
This should be in its own section under `### Breaking Changes` header and be a bit more descriptive so that people can more easily evaluate the breaking change and determine if they need to change anything or if they can safely upgrade without issue.